modules/telemetry-about-telemetry.adoc

```diff
@@ -14,6 +14,6 @@ Telemetry sends a carefully chosen subset of the cluster monitoring metrics to R
 * Limited usage information about {product-title} components and features
 * Summary info about alerts reported by the cluster monitoring component
 
-This continuous stream of data is used by Red Hat to monitor the health of clusters in real time and to react as necessary to problems that impact our customers. It also allows Red Hat to roll out {product-title} upgrades to customers so as to minimize service impact and continuously improve the upgrade experience.
+This continuous stream of data is used by Red Hat to monitor the health of clusters in real-time and to react as necessary to problems that impact our customers. It also allows Red Hat to roll out {product-title} upgrades to customers so as to minimize service impact and continuously improve the upgrade experience.
```

Comment:
We may want to link or connect customers to our upgrade documentation in this call out (if at all possible).

Reply:
@sferich888 thank you for your review!

This is a great suggestion. Since we cannot add cross-references within modules, I have now added an 'Additional resources' section under this module in the assembly. In that new section I have included a cross-reference to the update documentation.